tests/api/MessageV0/test_nic_00_out

```diff
@@ -94,6 +96,7 @@ messages {
         bypass_filtering: false
         type: VHOST_USER_SERVER
         path: "/tmp/qemu-vhost-nic-2"
+        packet_trace: true
```

Comment:
please, put protocol update, API server update, client update and test in separate commits